### PR TITLE
Cap Gun Crate

### PR DESCRIPTION
**Pull request by dge21**
## About the PR

Added a new crate that cargo can purchase containing a cap gun, two speed loaders, and two boxes of cartridges
## Why / Balance

The only way to get more cap gun cartridges is to buy a toy box for 900 spesos and only get six cartridges and a bunch of other shit you might not want to deal with. It's a massive waste of resources.
## Technical details

It's a small yml addition, so it should be easy enough to edit.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: 
- add: Added a new crate to delight all the clowns and mimes.

### CODE REVIEW

**Comment by FluffiestFloof** [comment]
These should be in the DeltaV folders and not in the Wizden files

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml [hunk 1]
```diff
@@ -368,4 +368,21 @@
...
+  name: cap gun box set
+  description: A box filled with fun, harmless cap guns that are sure to keep security entertained and the air smelling of sulfur.
+  id: CrateCapGunSet
+  parent: CratePlastic
+  components:
+  - type: StorageFill
+    contents:
+      - id: RevolverCapGun
+        amount: 1
+      - id: BaseSpeedLoaderCap
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Catalog/Fills/Crates/fun.yml]:
```suggestion
      - id: SpeedLoaderCap
```

**Comment by dge21** [comment]
> These should be in the DeltaV folders and not in the Wizden files

oh shit it should, I don't know how to change it without making a new pr tho

**Comment by dge21** [comment]
> These should be in the DeltaV folders and not in the Wizden files

oh shit you're right, I don't know how to change that without making a new pr though

**Review by VMSolidus — COMMENTED** [review]
You could probably also make them craftable at an Autolathe too. I don't see why anyone wouldn't. Check up on the cost of bullets from the autolathe, and compare it with equivalent amounts of plastic, since iirc capgun rounds are typically plastic shells with tiny amounts of flash powder in them.

**Comment by dge21** [comment]
Thats a good idea, I'm gonna redo this pr and make the changes in the proper folders

**Pull request closed without merging**